packages/ember-logger/lib/index.js

```diff
@@ -5,14 +5,14 @@ import { context } from 'ember-environment';
 function K() { return this; }
 
 function consoleMethod(name) {
```

Comment:
@krisselden 
Should I pull these methods `consoleMethod` and `assertPolyfill` into a util file?